packages/grafana-data/src/types/dashboard.ts

```diff
@@ -1,5 +1,34 @@
+import { FieldConfigSource } from './fieldOverrides';
+import { DataQuery, DatasourceRef } from './query';
+
+/** The scuemata version for the panel plugin */
+export type ModelVersion = [number, number];
+
 export enum DashboardCursorSync {
   Off,
   Crosshair,
   Tooltip,
 }
+
+export interface PanelModel<TOptions = any> {
```

Comment:
what do you think about using `unknown` instead to avoid developers making assumptions about untyped fields?

Reply:
I think it is worth trying -- I just plopped in `unknown` and typescript complains about some of the migrations.  Maybe we can try that in another PR.